brave-snouts-frontend/src/business/models/auction.model.ts

```diff
@@ -13,10 +13,14 @@ export class Auction {
     name: string;
     // basic auction description (TODO)
     description: string;
-    // start date and time when the auction starts
+    // start date  when the auction starts
     startDate: firebase.firestore.Timestamp;
-    // end date and time when the auction finishes
+    // start time when the auction start
+    startTime: firebase.firestore.Timestamp;
+    // end date when the auction finishes
     endDate: firebase.firestore.Timestamp;
+    // end time when the auction finishes
+    endTime: firebase.firestore.Timestamp;
```

Comment:
you can revert this to before since you are not using these props only date props which are constructed from time

Reply:
Still relevant